src/Datadog.Trace/Logging/DatadogLogging.cs

```diff
@@ -81,11 +81,6 @@ static DatadogLogging()
             {
                 // Don't let this exception bubble up as this logger is for debugging and is non-critical
             }
-            finally
-            {
-                // Log some information to correspond with the app domain
-                SharedLogger.Information(FrameworkDescription.Create().ToString());
```

Comment:
FYI @lucaspimentel and @colin-higgins since you have already approved, I wanted to point out this required change. By fixing `FrameworkDescription` to use the correct logger, it created a cycle between these two static constructors, so I'm removing this log line and delaying it to when it's actually constructed later in the Tracer. Let me know if you find that acceptable or have any suggestions.

Reply:
Yeah that's great, all for it.